website/react-magma-docs/src/components/MainNav/index.js

```diff
@@ -66,64 +124,50 @@ const ListItem = styled.li`
   padding: 0;
 `;
 
-const LinkStyles = props => `
-  align-items: center;
-  color: ${props.isInverse ? magma.colors.neutral600 : magma.colors.neutral700};
-  display: flex;
-  font-size: ${magma.typeScale.size02.fontSize};
-  justify-content: space-between;
-  line-height: ${magma.typeScale.size03.lineHeight};
-  padding: ${magma.spaceScale.spacing03} ${magma.spaceScale.spacing06};
-  text-decoration: none;
-`;
-
-const LinkHoverStyles = props => `
-background: ${
-  props.isInverse ? magma.colors.primary700 : magma.colors.neutral200
-};
-color: ${props.isInverse ? magma.colors.neutral600 : magma.colors.neutral700};
-`;
-
 const StyledLink = styled(Link)`
   ${LinkStyles};
-
-  &:hover,
-  &:focus {
+  &:hover {
     ${LinkHoverStyles}
   }
 `;
 
-const StyledExternalLink = styled.a`
-  ${LinkStyles};
-
-  &:hover,
-  &:focus {
-    ${LinkHoverStyles}
-  }
-`;
+const StyledActiveLink2 = {
+  content: "''",
+  position: ' absolute',
+  zIndex: ' 1',
+  top: ' 0',
+  left: ' 0',
+  height: ' 100%',
+  width: ' 4px',
+  borderRadius: ' 2px',
+  background: ` ${magma.colors.primary}`,
```

Comment:
Nitpick again, but there are extra spaces around these values. Can you try re-formatting the file?